extensions/choco/v1/template.json

```diff
@@ -0,0 +1,66 @@
+{
+   "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+   "contentVersion": "1.0.0.0",
+   "parameters": {
+		"artifactsLocation": {
+			"type": "string",
+			"minLength": 1,
+			"metadata": {
+				"description": "Artifacts Location - URL"
+			}
+		},
+		"apiVersionCompute": {
+			"type": "string",
+			"minLength": 1,
+			"metadata": {
+				"description": "Compute API Version"
+			}
+		},
+		"targetVMName":{
+			"type": "string",
+			"minLength": 1,
+			"metadata": {
+				"description": "Name of the vm to run the extension on"
+			}
+		},
+		"extensionParameters": {
+			"type": "securestring",
+			"minLength": 0,
+			"metadata": {
+				"description": "Custom Parameter for Extension - for Choco param is passed as extension parameter"
+			}
+		},
+		"vmIndex": {
+			"type": "int",
+			"metadata": {
+				"description": "index in the pool of the current agent, used so that we can get the extension name right"
+			}
+		}
+   },
+   "variables": {  },
+   "resources": [
+	{
+	  "apiVersion": "[parameters('apiVersionCompute')]",
+	  "dependsOn": [],
+	  "location": "[resourceGroup().location]",
+	  "type": "Microsoft.Compute/virtualMachines/extensions",
+	  "name": "[concat(parameters('targetVMName'),'/cse', parameters('vmIndex'))]",
```

Comment:
This should be named something more specific to chocolatey not to be confused with the already existing CSE